### PR TITLE
Updates, to ship the latest

### PR DESCRIPTION
**Pull request by matzew**
LEt's just ship the 0.14.2 bits

### CODE REVIEW

File: cmd/manager/kodata/knative-eventing/1-eventing.yaml [hunk 1]
```diff
@@ -622,12 +622,12 @@ spec:
           value: knative.dev/eventing
         - # PingSource
           name: PING_IMAGE
-          value: gcr.io/knative-releases/knative.dev/eventing/cmd/ping/adapter@sha256:034475c5983b63e38efad77b47f29d0d09e7a01bee861c4474255ed1d85bb48a
+          value: gcr.io/knative-releases/knative.dev/eventing/cmd/ping/adapter@sha256:c7272752928f6eeb9a66cf47c00b2d295ffb8517f2033dbbc8a5f461f6adafc2
         - name: JOB_RUNNER_IMAGE
-          value: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:knative-eventing-jobrunner
+          value: gcr.io/knative-releases/knative.dev/eventing/cmd/ping/jobrunner@sha256:b47877189b1e0f23c2617875574b16505251ae45ea091969332266621af99af8
```
Comment (aliok) [inline cmd/manager/kodata/knative-eventing/1-eventing.yaml]: wanna update the images?

**Comment by aliok** [comment]
/lgtm
/hold

Unhold if you wanted to release with gcr.io images quickly on purpose

**Comment by matzew** [comment]
Ah, yes - Ali !

thanks - will do

File: cmd/manager/kodata/knative-eventing/2-upgrade-to-v0.14.2.yaml [hunk 2]
```diff
@@ -15,6 +15,6 @@ spec:
       restartPolicy: Never
       containers:
       - name: upgrade-brokers
-        image: gcr.io/knative-releases/knative.dev/eventing/cmd/upgrade/v0.14.0@sha256:574b61b38c5bba600c6a00de39eb7d8b75bb38f551755c21eda938653b74bef3
+        image: gcr.io/knative-releases/knative.dev/eventing/cmd/upgrade/v0.14.0@sha256:8ad31868892f8730c818f23f45ea2fce278a37bc609954870f28f0bbfe3934ee
```
Comment (aliok) [inline cmd/manager/kodata/knative-eventing/2-upgrade-to-v0.14.2.yaml]: Gotta change this once the image is published I didn't see any references to gcr.io other than this

Comment (aliok) [reply cmd/manager/kodata/knative-eventing/2-upgrade-to-v0.14.2.yaml]: Couldn't find the correct image in https://github.com/openshift/release/blob/master/core-services/image-mirroring/knative/mapping_knative_v0_14_quay

**Pull request merged**